sac_test/src/lib.rs

```diff
@@ -0,0 +1,45 @@
+#![no_std]
```

Comment:
@dmkozh , you can delete `sac_test` from examples, I copied this to the other repo we wanted to use this contract within for integration tests, thanks for this sample code to demonstrate how auth next is enabled.